wlauto/external/uiauto/src/com/arm/wlauto/uiauto/BaseUiAutomation.java

```diff
@@ -272,4 +276,10 @@ public void setScreenOrientation(ScreenOrientation orientation) throws Exception
     public void unsetScreenOrientation() throws Exception {
         getUiDevice().unfreezeRotation();
     }
+
+   public void uiDevicePerformLongClick(UiObject view, int steps) throws Exception {
```

Comment:
Yes.  I think ALL of them need tidying up, especially as I think they are not all 5ms!  Agree about the general comment about tidying up.